R/convert-from-datacamp.R

```diff
@@ -1,44 +1,86 @@
-
-datacamp_to_learnr <- function(.repo) {
+#' datacamp_to_learnr
+#'
+#'
+#' @export
+datacamp_to_learnr <- function(cur_path=NULL, rename=NULL, author_name="Default", has_data=T) {
   # TODO: Check if working dir is a repo
   # TODO: Check if any uncommitted files are present
   # TODO: Check that the necessary files exist
   # TODO: Check if a "original materials" git branch exists
 
-# options(usethis.quiet = TRUE)
+options(usethis.quiet = TRUE)
```

Comment:
This would be better moved into something like `.onLoad()`, but I can take care of that.